clients/js/src/cmds/info/index.ts

```diff
@@ -1,22 +1,12 @@
 import yargs from "yargs";
-import * as chainId from "./chainId";
-import * as contract from "./contract";
-import * as emitter from "./emitter";
-import * as origin from "./origin";
-import * as registrations from "./registrations";
-import * as rpc from "./rpc";
-import * as wrapped from "./wrapped";
+import { YargsCommandModule } from "../Yargs";
+import { INFO_COMMANDS } from "./info";
 
 export const command = "info";
 export const desc = "Contract, chain, rpc and address information utilities";
 // Imports modules logic from root commands, more info here -> https://github.com/yargs/yargs/blob/main/docs/advanced.md#providing-a-command-module
 export const builder = (y: typeof yargs) =>
-  y
-    .command(chainId)
-    .command(contract)
-    .command(emitter)
-    .command(origin)
-    .command(registrations)
-    .command(rpc)
-    .command(wrapped);
+  // Commands can be imported as an array of commands.
+  // Documentation about command hierarchy can be found here: https://github.com/yargs/yargs/blob/main/docs/advanced.md#example-command-hierarchy-using-indexmjs
+  y.command(INFO_COMMANDS as unknown as YargsCommandModule[]);
```

Comment:
do we need the `as unkown` now? I see some others changed to remove the unknown intermediate

Reply:
Yes, I tried to get rid of `unknown` here, but typescript complains a lot about incompatible types.

On the root command this `unknown` is not necessary anymore, but looks like we still need to cast it here ( probably because this is a subcommand) 👀